gui/gui/inc/TGComboBox.h

```diff
@@ -143,6 +143,7 @@ class TGComboBox : public TGCompositeFrame, public TGWidget {
    virtual void Selected(Int_t widgetId, Int_t id);                  // *SIGNAL*
    virtual void Selected(Int_t id) { Emit("Selected(Int_t)", id); }  // *SIGNAL*
    virtual void Selected(const char *txt) { Emit("Selected(char*)", txt); } // *SIGNAL*
+   virtual void Selected() { Emit("Selected()"); } // *SIGNAL*
```

Comment:
Thanks for the PR, but I'm a bit reluctant to introduce a new, empty signal, which is counter-intuitive. I would prefer to be able to override it in the slot argument (i.e. by fixing the signal/slot classes)

Reply:
What about if we rename it to "Changed()" ? 

This way it would be less confusing, and also more meaningful as I only care when the entry changes, not what the actual value is ?

Reply:
Slightly better

Reply:
Done.